docs/docs/cloud/onboarding.md

```diff
@@ -85,11 +85,22 @@ While Meltano Cloud is in Beta, it may take up to one business day for your user
   <p>Installing the Meltano Cloud <a href="#prereq-2-provide-access-to-your-repo">GitHub App</a> to your organization is a separate process from granting Meltano Cloud access to use your GitHub profile for login purposes. Both the GitHub App installation for your organization <em>and</em> the OAuth grant flow for your profile must be performed in order to have full access to Meltano Cloud functionality.</p>
 :::
 
-### Step 3: Set default project and validate access and functionality
+### Step 3: Create a Project
```

Comment:
@tayloramurphy -- is this sufficient for the onboarding docs update here?

Reply:
Here's the preview: https://deploy-preview-7620--meltano.netlify.app/cloud/onboarding

Reply:
Looks good!